app/views/saved_declarations.scala.html

```diff
@@ -39,20 +38,27 @@
 
     <h1>@messages("saved.declarations.title")</h1>
 
+    @helper.CSRF.formField
```

Comment:
I'm not sure that CSRF token is necessary here

Reply:
Seems like I have copied it and pasted it in the wrong page.
This has been addressed.